include/mbgl/shaders/collision_layer_ubo.hpp

```diff
@@ -14,8 +14,8 @@ struct alignas(16) CollisionUBO {
 static_assert(sizeof(CollisionUBO) % 16 == 0);
 static_assert(sizeof(CollisionUBO) == 80);
 
-typedef CollisionUBO CollisionBoxUBO;
-typedef CollisionUBO CollisionCircleUBO;
+using CollisionUBO CollisionBoxUBO;
```

Comment:
syntax: `using CollisionBoxUBO = CollisionUBO;`

Reply:
Yes, sure. I was focused on something else when I corrected this. Now should be fine. 😄